Makefile

```diff
@@ -37,6 +37,20 @@ production:
 	$(eval ENV_SHORT=pd)
 	$(eval ENV_TAG=prod)
 
+.PHONY: review
+review:
+	$(if $(pr_id), , $(error Missing environment variable "pr_id"))
+	$(eval DEPLOY_ENV=review)
+	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-development)
+	$(eval RESOURCE_NAME_PREFIX=s165d01)
+	$(eval ENV_SHORT=rv)
+	$(eval ENV_TAG=rev)
+	$(eval env=review-pr-$(pr_id))
+	$(eval backend_config=-backend-config="key=review/$(env).tfstate")
+	$(eval export TF_VAR_apply_qts_app_name=apply-for-qts-in-england-$(env))
```

Comment:
Probably these TF_VARs not needed since we build these names using `DEPLOY_ENV` value and tf local vars should build these names dynamically.  Is there any other reason for passing these vars from Makefile ?

Reply:
See above https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/189#discussion_r914640527

Reply:
Can we pass only 1 variable like "app_suffix" instead? and build the app name, pg name and redis name inside of terraform